documentation/documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/python_evaluator.md

```diff
@@ -0,0 +1,12 @@
+# Python evaluator
+## General usage
+This evaluator is responsible for running and executing tests on a student's Python code.
```

Comment:
these files don't sound very Dutch

Reply:
I'll translate the runner files in another PR since i don't have these pages finished and the PR will get too large

Reply:
Ok, I'll make an issue for it before closing this PR

Reply:
Okay that's perfect, thanks